### DISCUSSION TITLE
Give us the ability to use more complex Markdown abilities like footnotes

### DISCUSSION THREAD

**Discussion by identithree** [community/community#2907] [Discussions]
It would be really cool if we could have footnotes in not only Discussions but also in the GitHub markdown editor

**Comment by becca** [comment]
What are you hoping to use this for? Do you have any examples? 👀

**Reply by identithree** [reply]
Yes, apologies for not getting back in a timely manner. If I have a SECURITY.md file, I would like to be able to put a footnote on one of the supported versions that says "Alpha not supported"

**Comment by markcmiller86** [comment]
I saw this and wanted to mention [this approach](https://github.com/betterscientificsoftware/bssw.io/blob/master/Articles/Blog/ReferencesInMarkdownHybridApproach-wikized.md) which is aimed more at web-page publication but also does work with pure GFM on GitHub in a browser.
